mitodl/micromasters
# Use site_key in discussions JWT token

#### What are the relevant tickets?
Closes mitodl/open-discussions#444
Requires mitodl/open-discussions#492 (either merged or checkout that branch while testing this)

#### What's this PR do?
Switches MM to use site_key for site data, removes passing of old urls.

#### How should this be manually tested?
- In `.env` set `OPEN_DISCUSSIONS_SITE_KEY` to a value matching a `AuthenticatedSite` record. See [this PR](https://github.com/mitodl/open-discussions/pull/492) for instructions on how to set that up
- Verify you can still authenticate to OD and see the correct title and urls in OD
